twoopstracker/twoops/views.py

```diff
@@ -248,6 +255,8 @@ def get_queryset(self):
             start_date = datetime.date.fromisoformat(start_date)
         if end_date:
             end_date = datetime.date.fromisoformat(end_date)
+        if category:
+            tweets = tweets.filter(owner__category__name=category)
```

Comment:
Not sure this will yield correct results or not. See this [SO](https://stackoverflow.com/a/28253623) about `.filter(A, B)` vs `.filter(A).filter(B)`

Reply:
@kilemensi  I think for us, we need the `.filter(A).filter(B)` because we want to have all the previously filtered tweets and filter them further.

Reply:
If you’re sure @esirK